bears/python/PEP8NotebookBear.py

```diff
@@ -84,13 +85,17 @@ def run(self, filename, file,
         will not change functionality of the code in any way.
 
         :param max_line_length:   Maximum number of characters for a line.
+                                  When set to 0 allows infinite line length.
         :param indent_size:       Number of spaces per indent level.
         :param pep_ignore:        A list of errors/warnings to ignore.
         :param pep_select:        A list of errors/warnings to exclusively
                                   apply.
         :param local_pep8_config: Set to true if autopep8 should use a config
                                   file as if run normally from this directory.
         """
+        if not max_line_length:
+            max_line_length = sys.maxsize
+
```

Comment:
Is this extra line really necessary?

Reply:
I think yes because line no. 120 in this program has also the same concept . It has also the gap of one line .

Reply:
> @Naveenaidu I see the test of both the PR .In both there is a function made for the test but the test file for this i.e https://github.com/coala/coala-bears/blob/master/tests/python/PEP8NotebookBearTest.py I am unable to understand In which language it is written. I have see this type of python program first time with meta data.

@rishabhgarg25699 Python has been used to write the test file. It's similiar to GnuIndentBearTest. It would also help you if you understand what the role of `PEP8NotebookBear` is. Just go through the [doc-string](https://github.com/coala/coala-bears/blob/559c953e1b5787df9dddf96ece8449e50c945716/bears/python/PEP8NotebookBear.py#L14) of the file, the test file will make more sense to you.

> I have also to made only function for test at last of program ?

Please try to understand the process. You could actually get enough idea on writing the tests on seeing the two PR which have been attached.
I'll give you few hints:-
1. You need to make a file which has more than 80 characters. Have a look at the `source field` in the `good file`.
2. Once you create the above file you can then create the function( take reference from the PR's mentioned by @newbazz in the issue) in which you will have to append the settings of maxlength and voila you would have successfully written your tests. :)

Hope you could solve this issue now. Good Luck :)

Reply:
@Naveenaidu I have to add code in good_file as well as in bad_file and then at last i have to add a function  in the test i.e https://github.com/coala/coala-bears/blob/master/tests/python/PEP8NotebookBearTest.py

Reply:
I have to add this 
{
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "a markdown cell is not a code cell"
   ]
  },

in good_file

Reply:
@rishabhgarg25699 That's for you to figure it out :) There won't be much fun left for you if I do tell.
A little suggestion, which I follow myself. Why not try out your approach in your local computer first, in this way you would not have to wait for someone to look at your approach and tell you if it's right or wrong. This will also help you to eliminate your approaches and arrive at a right solution. And moreover it will also help you to ask the right questions :)